packages/react-components/react-tags/src/components/TagButton/useTagButtonStyles.ts

```diff
@@ -1,20 +1,46 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { TagButtonSlots, TagButtonState } from './TagButton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const tagButtonClassNames: SlotClassNames<TagButtonSlots> = {
   root: 'fui-TagButton',
-  // TODO: add class names for all slots on TagButtonSlots.
-  // Should be of the form `<slotName>: 'fui-TagButton__<slotName>`
+  contentButton: 'fui-TagButton__contentButton',
+  avatar: 'fui-TagButton__avatar',
+  icon: 'fui-TagButton__icon',
+  primaryText: 'fui-TagButton__primaryText',
+  secondaryText: 'fui-TagButton__secondaryText',
+  dismissButton: 'fui-TagButton__dismissButton',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'inline-flex',
   },
+  contentButton: {
```

Comment:
Not a blocker now but these styles look identical to the ones in `useTagStyles`. Would be nice to find a way to share them.

Reply:
agreed :D we'll see where they land in the end